sklearn/linear_model/tests/test_logistic.py

```diff
@@ -492,6 +493,37 @@ def test_logistic_cv():
     assert_array_equal(scores.shape, (1, 3, 1))
 
 
+@pytest.mark.parametrize('scoring', ['accuracy', 'f1', 'neg_log_loss',
+                                     'precision', 'recall'])
+def test_logistic_cv_multinomial_score(scoring):
+    # test that LogisticRegressionCV uses the right score to compute its
+    # cross-validation scores when using a multinomial scoring
+    # see https://github.com/scikit-learn/scikit-learn/issues/8720
+    X, y = make_classification(n_samples=100, random_state=0, n_classes=3,
+                               n_informative=6)
+    train, test = np.arange(80), np.arange(80, 100)
+    lr = LogisticRegression(C=1., solver='lbfgs', multi_class='multinomial')
+    # we use lbfgs to support multinomial
+    params = lr.get_params()
+    # we store the params to set them further in _log_reg_scoring_path
+    for key in ['C', 'n_jobs', 'warm_start']:
+        del(params[key])
+    lr.fit(X[train], y[train])
+    if scoring in ['f1', 'precision', 'recall']:
```

Comment:
I don't get what's necessitated this if-else. I think this would be less confusing if you just used names like `precision_macro` in parametrization.

Also, `f1_micro == precision_micro == recall_micro == accuracy` in the multiclass case, so these are redundant.

Reply:
alright, I will try to include this in the parametrization and remove redundancy